api/src/main/java/org/openmrs/module/idgen/service/BaseIdentifierSourceService.java

```diff
@@ -425,6 +425,14 @@ public void checkAndRefillIdentifierPool(IdentifierPool pool){
     public IdentifierSource getIdentifierSourceByUuid(String uuid) {
         return dao.getIdentifierSourceByUuid(uuid);
     }
+    
```

Comment:
do we need these comments

Reply:
Yes this specific file is very well commented.